src/features/Planner/data/TaskApi.js

```diff
@@ -11,17 +14,25 @@ const axios = BaseAxios.create({
 const TaskApi = {
     createList: (name, clientId, fromId) =>
         promiseFlux(
-            axios.post(`/`, {
-                name,
-                fromId,
```

Comment:
Guess which one of me missed a use-case on the graphql? Me! Opened #158 for this.